static/js/stream_edit_subscribers.js

```diff
@@ -305,6 +307,43 @@ function update_subscribers_list_widget(subscriber_ids) {
     subscribers_list_widget.replace_list_data(users);
 }
 
+export function rerender_subscribers_list(sub) {
+    if (!hash_util.is_editing_stream(sub.stream_id)) {
+        blueslip.info("ignoring subscription for stream that is no longer being edited");
+        return;
+    }
+
+    if (sub.stream_id !== current_stream_id) {
+        // This should never happen if the prior check works correctly.
+        blueslip.error("current_stream_id does not match sub.stream_id for some reason");
+        return;
+    }
+
+    if (!stream_data.can_view_subscribers(sub)) {
+        return;
+    }
+
+    const user_ids = peer_data.get_subscribers(sub.stream_id);
+    const user_can_remove_subscribers = stream_data.can_unsubscribe_others(sub);
+    const $parent_container = stream_settings_containers
+        .get_edit_container(sub)
+        .find(".edit_subscribers_for_stream");
+
+    $parent_container.html(
+        render_stream_members({
+            can_access_subscribers: true,
+            can_remove_subscribers: user_can_remove_subscribers,
+            render_subscribers: sub.render_subscribers,
+        }),
+    );
+    subscribers_list_widget = make_list_widget({
+        $parent_container,
+        name: "stream_subscribers",
+        user_ids,
+        user_can_remove_subscribers,
+    });
```

Comment:
This is a bit messy -- probably the ideal version of this would share this block of code with the original rendering of the stream edit UI. But I'm OK merging this for now.

Reply:
Oh I guess it does call `make_list_widget`; idunno maybe there's not much more to share.